usr/src/uts/common/os/kmem.c

```diff
@@ -2185,24 +2186,46 @@ kmem_depot_ws_zero(kmem_cache_t *cp)
 }
 
 /*
+ * Setting it this high (1GB) casues the explicit kpreempt to have
+ * very little effect.  Setting it to 1MB causes us to call
+ * kpreempt() approximately every 2 milliseconds.
```

Comment:
This comment sounds like something that will be out of date (if we're lucky) or misleading (if we're not) in a year.